### PR TITLE
AUI-2099 Refactor A.FormBuilderFieldList._uiSetFields

### PR DESCRIPTION
**Pull request by fernandosouza**
Hi @mairatma. I'm sending it directly to you because Java already reviewed in my laptop. Please. I need it to keep working on a Liferay Portal task.

### CODE REVIEW

File: src/aui-form-builder/js/aui-form-builder-field-list.js [hunk 1]
```diff
@@ -107,6 +107,18 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
     },
 
     /**
+     * Executed to the last empty column.
+     *
+     * @method _afterAppendInAEmptyColumn
+     * @param {Node} content
+     * @protected
+     */
+    _afterAppendInAEmptyColumn: function(content) {
```
Comment (fernandosouza) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: @mairatma. Please, help me to find a better name to this method.

**Comment by mairatma** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-builder/js/aui-form-builder-field-list.js (hunk 1 above)
Comment (mairatma) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: Could be something like `addEmptyColumnFieldClasses_`

Comment (mairatma) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: Oops, I ended up merging without waiting for the rename, forgot about it :)
Could you send a separate pr just with this in that case?

Comment (fernandosouza) [inline src/aui-form-builder/js/aui-form-builder-field-list.js]: Sure, @mairatma. Almost done.
